src/components/TaskPreviewPane.tsx

```diff
@@ -0,0 +1,118 @@
+import React, { useEffect, useRef } from 'react';
+import { useSelector, useDispatch } from 'react-redux';
+import { FlatList, StyleSheet, Animated, Dimensions } from 'react-native';
+import { Text } from '@ui-kitten/components';
+import {
+    calendarCardDimensions,
+    headerBarHeight,
+    bottomTabHeight,
+    windowHeight,
+} from '../shared/constants';
+import { LinearGradient } from 'expo-linear-gradient';
+import { TaskItem } from './ListItems/TaskItem';
+import { PanGestureHandler, State as GestureState } from 'react-native-gesture-handler';
+import { hidePreviewPane } from '../store/actions';
+import { coloredBackgroundGradient1, coloredBackgroundGradient2 } from '../ui/colors';
+
+const calendarHeight: number = calendarCardDimensions.totalHeight;
+// const windowHeight = Dimensions.get('window').height;
+const taskPreviewHeight = windowHeight - calendarHeight + headerBarHeight;
+const closingHeight = windowHeight - bottomTabHeight - 60; //not sure why i need this 60 but it doesnt work well on my phone without it, maybe windowheight doesnt include the header bar height??
```

Comment:
weird. not sure if this SO link makes sense: https://stackoverflow.com/questions/54364798/react-native-how-to-get-the-complete-height-of-the-device-when-full-screen-mo